ext/js/data/sandbox/anki-note-data-creator.js

```diff
@@ -867,10 +871,20 @@ function getCloze(dictionaryEntry, context) {
     if (typeof text !== 'string') { text = ''; }
     if (typeof offset !== 'number') { offset = 0; }
 
+    const textSegments = [];
+    for (const {text: text2, reading: reading2} of distributeFuriganaInflected(term, reading, text.substring(offset, offset + originalText.length))) {
+        if (reading2 !== '') {
```

Comment:
ternary
`textSegments.push(reading2.length > 0 ? reading2 : text2)`

(length vs '' check probably doesn't matter, so choose whichever you prefer)

Reply:
Length does feel a bit more appropriate here. Added this change.